frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java

```diff
@@ -33,7 +33,7 @@ public GitMachetePanel(Project project) {
     super(/* vertical */ false, /* borderless */ true);
     LOG.debug("Instantiating");
 
-    var selectionComponent = project.getService(SelectedVcsRepositoryProvider.class).getSelectionComponent();
```

Comment:
I think now two things can be simplified:

1) `graphTable` no longer needs to be a field of this class (subject to small reworks around `createGitMacheteVerticalToolbar`)
2) `graphTable` in uiTests/src/test/resources/scenarios-and-utils.rhino.js (lines 75+) can possibly get retrieved via `project.getService(GraphTableProvider).getGraphTable()` instead of getting extracted via `tab.getComponent().getGraphTable()`

Reply:
(subject to `importClass` on `GraphTableProvider`)